docs/podman-run.1.md

```diff
@@ -523,7 +523,7 @@ If you omit the unit, the system uses bytes. If you omit the size entirely, the
 
 **--sig-proxy**=*true*|*false*
 
-Proxy received signals to the process (non-TTY mode only). SIGCHLD, SIGSTOP, and SIGKILL are not proxied. The default is *true*.
+Proxy received signals in the client to the container process. SIGCHLD, SIGSTOP, and SIGKILL are not proxied. The default is *true*.
```

Comment:
I'm just wondering if people might get hung up on "what's the client?".  Could we simplify this to "Proxy all received signals to the container process"  This is what docker does, less 'container'.

Reply:
I actually did not like that, since I had no idea what that meant.  In the docker case they are talking about the docker CLI process proxyinging the signal to the daemon and then onto the container. In our case we are doing the same.  
I am trying to say if you send a signal to the podman process, it proxies the signal to pid1 of the container,  but PID1 is a little technical for most people.

Reply:
I'm OK with it the way @rhatdan has it, even though I prefer the other wording.  I'm OK with merging this as is if @mheon is hip.  If we find that it's confusing, we can always change it later.

Reply:
How about "Proxy signals received by Podman to the container process"?

Reply:
Just podman might not be descriptive enough... 'this podman command'? 'podman run'?